src/Apps/Artwork/Components/PricingContext.tsx

```diff
@@ -50,14 +51,37 @@ export class PricingContext extends React.Component<PricingContextProps> {
     // I'm just for tracking!
   }
 
+  @track({
+    action_type: Schema.ActionType.Click,
+    flow: Schema.Flow.ArtworkPriceContext,
+    subject: Schema.Subject.BrowseWorks,
+    type: Schema.Type.Chart,
+  })
+  collectPageLinkClick(dimension, category, sizeScore, artistId) {
+    const url = createCollectUrl({ dimension, category, sizeScore, artistId })
+    if (typeof window !== "undefined") {
+      window.open(url)
+    }
+  }
+
   // TODO: Investigate why metaphysics is returning null instead of zero for minPrice
   render() {
     const { artwork } = this.props
-
     if (!artwork.pricingContext) {
       return null
     }
 
+    let sizeScore
+
+    if (artwork.sizeScore != null) {
```

Comment:
I don't think we need to be concerned with sizeScore in reaction yet. As mentioned elsewhere in this review I think we can lean on MP and vortex to return us a meaningful `appliedFilters.dimension`.